### PR TITLE
[Vision & Multimodal Model Asset release] Vision Model MlFlow Conversion Updates

### PR DESCRIPTION
**Pull request by KonakanchiSwathi**
1/ Update the mlflow version
2/ Add save_image functionality
3/ Update env versions

### CODE REVIEW

**Review by rjaincc — DISMISSED** [review]

File: assets/training/model_management/components/convert_model_to_mlflow/spec.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
 
 name: convert_model_to_mlflow
-version: 0.0.26
+version: 0.0.29
```
Comment (rjaincc) [inline assets/training/model_management/components/convert_model_to_mlflow/spec.yaml:4]: Why are we bumping the version to 29 for this component?

Comment (KonakanchiSwathi) [reply assets/training/model_management/components/convert_model_to_mlflow/spec.yaml:4]: Iam regisering in preview-test1 to test models

**Review by rjaincc — DISMISSED** [review]

File: assets/common/components/delete_endpoint/spec.yaml [hunk 2]
```diff
@@ -8,9 +8,7 @@ is_deterministic: True
 display_name: Delete Endpoint
 description:
   Deletes an endpoint resource.
-
 environment: azureml://registries/azureml/environments/python-sdk-v2/versions/15
-
```
Comment (rjaincc) [inline assets/common/components/delete_endpoint/spec.yaml:13]: why are we removing empty lines? Please revert it, if not needed.

File: assets/common/components/deploy_model/spec.yaml [hunk 3]
```diff
@@ -8,9 +8,7 @@ is_deterministic: True
 display_name: Deploy model
 description:
   Deploy a model to a workspace. The component works on compute with [MSI](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-manage-compute-instance?tabs=python) attached.
-
 environment: azureml://registries/azureml/environments/python-sdk-v2/versions/15
-
```
Comment (rjaincc) [inline assets/common/components/deploy_model/spec.yaml:13]: why are we removing empty lines? Please revert it, if not needed.

File: assets/training/model_management/components/convert_model_to_mlflow/spec.yaml (hunk 1 above)
Comment (rjaincc) [reply assets/training/model_management/components/convert_model_to_mlflow/spec.yaml:4]: This version is picked up for releasing into prod. If there are no changes, please don not update the version.

Comment (rjaincc) [reply assets/training/model_management/components/convert_model_to_mlflow/spec.yaml:4]: This versions will be used for release in prod registry. If the version bump is not required for prod, please don't bump the version up.

Comment (KonakanchiSwathi) [reply assets/training/model_management/components/convert_model_to_mlflow/spec.yaml]: reverted the version

File: assets/common/components/deploy_model/spec.yaml (hunk 3 above)
Comment (KonakanchiSwathi) [reply assets/common/components/deploy_model/spec.yaml]: reverted

File: assets/common/components/delete_endpoint/spec.yaml (hunk 2 above)
Comment (KonakanchiSwathi) [reply assets/common/components/delete_endpoint/spec.yaml:13]: reverted

File: assets/common/components/delete_endpoint/spec.yaml [hunk 4]
```diff
@@ -1,40 +0,0 @@
-$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
```
[the remark this answers is not in the document]
Comment (KonakanchiSwathi) [reply assets/common/components/delete_endpoint/spec.yaml:1]: yes i want to remove it from this PR

**Review by rjaincc — APPROVED** [review]

**Pull request merged**